lib/timetable-form/Form.tsx

```diff
@@ -169,6 +221,13 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
           <Button>Export Timetable Header CSV</Button>
         </a>
       </div>
+      <Button
```

Comment:
I have added this button here to showcase the validation as we don't have a submit button at the moment (or anything to submit the form to)

Reply:
alternatively the validation could be triggered when exporting the csv?

Reply:
This is fine for now, that's a question for design